pkg/cmd/clean/exec.go

```diff
@@ -63,6 +65,17 @@ func (o *Options) Run() error {
 		return err
 	}
 
+	// check if any managed cluster exist or not
+	exist, err := isManagedClusterExist(config)
+	if err != nil {
+		return err
+	}
+
+	if exist {
+		fmt.Fprintf(o.Streams.Out, "Please detach the managed clusters from the hub control plane\n")
```

Comment:
Is it better to print the specific managed clusters' name? or `Please detach **all** managed clusters ...`

Reply:
I think `all managed clusters` is better.